hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmBucketReadWriteFileOps.java

```diff
@@ -149,6 +155,11 @@ public Void call() throws Exception {
     contentGenerator = new ContentGenerator(fileSizeInBytes, bufferSize);
     timer = getMetrics().timer("file-create");
 
+    mbs = ManagementFactory.getPlatformMBeanServer();
+    bean = new ObjectName(
```

Comment:
@tanvipenumudy after reading this mbean output, now the metric value stores a combined read/write lock held/waiting time for all the resources. For example, we can't differentiate whether BUCKET_LOCK is consuming long time than the fine grained KEY_PATH_LOCK.

Probably, we need to introduce resource specific metrics to get more granular view about the exact resource(BUCKET, KEY_PATH etc) consumption samples, isn't it?

I've added this comment here to track/remember this point. IMHO, please raise a separate jira sub-task to discuss and provide resource specific metrics values. We can come back to this task once we finish the basic structure.

Reply:
Noted, I agree to this! Thank you for the review!

Reply:
Please find the link to the separate JIRA sub-task: https://issues.apache.org/jira/browse/HDDS-6712, thanks.